webpack.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = () => {
       path: path.resolve(__dirname, 'dist'),
       publicPath: '/',
     },
+    devtool: isDevelopment ? 'source-map' : false,
```

Comment:
I'm curious about this line. what is it for?

Reply:
this option will help you generate source-map for debugging, you can refer the link below for more details
https://webpack.js.org/configuration/devtool/

**without `devtool` option**:
![Screenshot 2022-08-21 220254](https://user-images.githubusercontent.com/28136235/185797569-747ec750-5673-470a-bb10-dd01650fcf23.png)


**with `devtool` option:**
![Screenshot 2022-08-21 220359](https://user-images.githubusercontent.com/28136235/185797586-c4ea8b45-cec0-4656-966f-9ca5dca26613.png)